src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StudentAssessmentFact/MSSQL/v_3_3/0001_StudentAssessmentFact_should_match_column_dictionary.xml

```diff
@@ -43,6 +43,10 @@
         <ColumnName>StudentAssessmentIdentifier</ColumnName>
         <DataType>nvarchar</DataType>
     </Result>
+   <Result>
+        <ColumnName>StudentUniqueId</ColumnName>
```

Comment:
The new column is StudentKey

Reply:
Additionally, you changed this file, which is correct. But you forgot to change the same file for DataStandard 3.1 and 3.2.

Reply:
Done.